fuseki/sample-ubd.trig

```diff
@@ -0,0 +1,280 @@
+    cc:csvColumnSample "3.4" ;
+    cc:csvColumnSample "2.1" ;
+    cc:csvColumnSample "2.9" ;
+  .
+
+  <cube-project/ubd/csv-source/ubd/column/aggregation>
+    a csvw:Column ;
+    schema:name "AGGREGATION" ;
+    dtype:order 3 ;
+    cc:csvColumnSample "13.4" ;
+    cc:csvColumnSample "14.4" ;
+    cc:csvColumnSample "18.0" ;
+  .
+}
+
+<cube-project/ubd/csv-source/stations> void:inDataset <ubd-example> .
+graph <cube-project/ubd/csv-source/stations> {
+  <cube-project/ubd/csv-source/stations>
+    a cc:CSVSource, hydra:Resource ;
+    cc:csvMapping <cube-project/ubd/csv-mapping> ;
+    schema:associatedMedia  [ a  schema:MediaObject ;
+                              schema:contentUrl <http://s3:9000/cube-creator/test-data/ubd28/input_CH_yearly_air_immission_basetable.csv> ;
+                              schema:identifier "test-data/ubd28/input_CH_yearly_air_immission_basetable.csv";
+                            ] ;
+    schema:name "stations.csv" ;
+    csvw:dialect <cube-project/ubd/csv-source/stations/dialect> ;
+    csvw:column <cube-project/ubd/csv-source/ubd/column/year> ;
+    csvw:column <cube-project/ubd/csv-source/stations/column/station-name-fr> ;
+    csvw:column <cube-project/ubd/csv-source/stations/column/station-name-de> ;
+  .
+
+  <cube-project/ubd/csv-source/stations/dialect>
+    csvw:quoteChar "\"" ;
+    csvw:delimiter "," ;
+    csvw:header    true ;
+  .
+
+  <cube-project/ubd/csv-source/stations/column/station-id>
+    a csvw:Column ;
+    schema:name "STATION_ID" ;
+    dtype:order 0 ;
+    cc:csvColumnSample "1" ;
+    cc:csvColumnSample "2" ;
+    cc:csvColumnSample "3" ;
+  .
+
+  <cube-project/ubd/csv-source/stations/column/station-name-fr>
+    a csvw:Column ;
+    schema:name "STATION_NAME_FR" ;
+    dtype:order 1 ;
+    cc:csvColumnSample "ABC" ;
+    cc:csvColumnSample "DEF" ;
+    cc:csvColumnSample "GHI" ;
+  .
+
+  <cube-project/ubd/csv-source/stations/column/station-name-de>
+    a csvw:Column ;
+    schema:name "STATION_NAME_DE" ;
+    dtype:order 2 ;
+    cc:csvColumnSample "ABK" ;
+    cc:csvColumnSample "DEF" ;
+    cc:csvColumnSample "GHI" ;
+  .
+}
+
+<cube-project/ubd/csv-mapping/tables> void:inDataset <ubd-example> .
+graph <cube-project/ubd/csv-mapping/tables> {
+  <cube-project/ubd/csv-mapping/tables>
+    a cc:TableCollection, hydra:Collection ;
+    hydra:title "Tables" ;
+    hydra:manages [ hydra:property rdf:type ;
+                    hydra:object   cc:Table
+                  ] ;
+    hydra:manages [ hydra:property cc:csvMapping ;
+                    hydra:object   <cube-project/ubd/csv-mapping>
+                  ] ;
+    cc:csvMapping <cube-project/ubd/csv-mapping> ;
+  .
+}
+
+<project/ubd/csv-mapping/table-observation> void:inDataset <ubd-example> .
+graph <project/ubd/csv-mapping/table-observation> {
+  <project/ubd/csv-mapping/table-observation>
+    a cc:Table, cc:ObservationTable, hydra:Resource ;
+    cc:csvw <project/ubd/csv-mapping/table-observation/csvw> ;
```

Comment:
This is something which was missing from #10 but will be needed to get the pipeline running